zeus/monitor/energy.py

```diff
@@ -36,18 +38,22 @@ class Measurement:
 
     Attributes:
         time: Time elapsed (in seconds) during the measurement window.
-        energy: Maps GPU indices to the energy consumed (in Joules) during the
+        gpu_energy: Maps GPU indices to the energy consumed (in Joules) during the
             measurement window. GPU indices are from the DL framework's perspective
             after applying `CUDA_VISIBLE_DEVICES`.
+        cpu_energy: CPU energy consumption (in J) during the measurement window.
+        dram_energy: DRAM energy consumption (in J) during the measurement window.
     """
 
     time: float
-    energy: dict[int, float]
+    gpu_energy: dict[int, float]
+    cpu_energy: dict[int, float]
+    dram_energy: Optional[dict[int, float]] = None
 
     @cached_property
     def total_energy(self) -> float:
         """Total energy consumed (in Joules) during the measurement window."""
-        return sum(self.energy.values())
+        return sum(self.gpu_energy.values())
```

Comment:
Ah shoot okay, let's leave this alone for now because I think chaning this to `total_gpu_energy` is potentially a **big** breaking change to our `examples/`, but this has to be updated soon. Please note this as a TODO.